cmake/common.cmake

```diff
@@ -539,6 +539,7 @@ if(NOT ${TARGET} STREQUAL "virtual")
     ccfcrypto.host
     evercrypt.host
     CURL::libcurl
+    secp256k1.host
```

Comment:
Necessary now that `host/main.cpp` parses members' certificates using `keypair.h`. I know it's somehow unlikely that members' certs will use that (fast) curve but it keeps things neat for now. Happy to create a parsing-only file that only depends on `mbedtls` if you think it's better.